src/EnergyPlus/EvaporativeFluidCoolers.cc

```diff
@@ -978,9 +978,9 @@ namespace EvaporativeFluidCoolers {
 
             this->InitEvapFluidCooler();
             this->SizeEvapFluidCooler();
-            MinLoad = 0.0; // signifies non-load based model (i.e. forward
-            MaxLoad = 0.0; // heat exhanger model)
-            OptLoad = 0.0;
+            MinLoad = 0.0; // signifies non-load based model (i.e. forward heat exhanger model)
+            MaxLoad = this->HighSpeedStandardDesignCapacity * this->HeatRejectCapNomCapSizingRatio;
+            OptLoad = this->HighSpeedStandardDesignCapacity;
```

Comment:
Interesting, this has been zero since **at least** the 8.2 version when we came from Fortran.  I guess maybe since the very beginning of this code?  This does seem quite reasonable.

Reply:
@Myoldmopar Wow--that's interesting...maybe a little scary.  But I'm glad @EnergyArchmage found this and that it got fixed.